packages/dnspyex.vm/tools/chocolateyinstall.ps1

```diff
@@ -2,12 +2,22 @@ $ErrorActionPreference = 'Stop'
 Import-Module vm.common -Force -DisableNameChecking
 
 try {
-  $toolName = 'dnSpyEx'
+  $toolName = 'dnSpy'
```

Comment:
I think this should stay `dnSpyEx` to indicate we're using the updated tool.

Reply:
Not really, the tool name (the exe) is called dnSpy.

The only problem is if we want to install dnSpy (the not Ex version), but it is not easy as we need to compile it. I was thinking maybe it is enough with dnSpyEx and in this case we can call the tool `dnSpy`.

In case you think we should call the tool `dnSpyEx`, how do you propose we implement it?

Reply:
Ah, nevermind. Confused this with the package name I guess :)